WordPressAuthenticator.podspec

```diff
@@ -45,4 +45,8 @@ Pod::Spec.new do |s|
   # If you want to update which of these is used, specify it in the host app.
   s.dependency 'WordPressKit', '~> 4.18-beta' # Don't change this until we hit 5.0 in WPKit
   s.dependency 'WordPressShared', '~> 1.12-beta' # Don't change this until we hit 2.0 in WPShared
+
+  # Fixing arm64 issue with Xcode 12: https://github.com/CocoaPods/CocoaPods/issues/10104
+  s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

Comment:
This change makes this pod incompatible with Xcode on Apple silicon 🤔 

Did the CocoaPods update make this unnecessary?

Reply:
> Did the CocoaPods update make this unnecessary?

Unfortunately not 😢 

There end up being linker errors during linting:

```
    ld: in /var/folders/jb/jdc8rjfx4sx60n_0xnclhtc00000gn/T/CocoaPods-Lint-20201211-68458-riqq0i-WordPressAuthenticator/Pods/GoogleSignIn/Frameworks/GoogleSignIn.framework/GoogleSignIn(GIDEMMErrorHandler_3a47e13d8ca81b41e9cdb7ef5468004a.o), building for iOS Simulator, but linking in object file built for iOS, file '/var/folders/jb/jdc8rjfx4sx60n_0xnclhtc00000gn/T/CocoaPods-Lint-20201211-68458-riqq0i-WordPressAuthenticator/Pods/GoogleSignIn/Frameworks/GoogleSignIn.framework/GoogleSignIn' for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
```